ports/libgit2/vcpkg.json

```
@@ -30,6 +29,9 @@
         "mbedtls"
       ]
     },
+    "openssh": {
+      "description": "SSH support (OpenSHH)"
```

```suggestion
      "description": "SSH support (OpenSSH)"
```

There is no dependency here. Is this using an external program (seeing the `exec` in the portfile`) instead of a lib?

---

fixed the typo: replaced with a more clear feature descriuption